components/Layout/popUp/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
+import { FC } from 'react'; 
+import { Form } from '../MsgForm/index'
+
+export const PopUp: FC = () => {
+  return(
+    <Popup trigger={<button> Questions</button>} >
```

Comment:
make it like a function and try to don't pass buttons or JSX as props 

- rewrite this

Reply:
no need for `reactjs-popup` ... please write your own modal